packages/docs/pages/connections/flickr.md

```diff
@@ -0,0 +1,21 @@
+# Flickr
+
+:::info This page explains the steps you need to follow to set up the Flickr
```

Comment:
The generic description should be started after the third line otherwise, it breaks the format of the info box.

Reply:
Right! It's just that my vscode automatically formats it like this on save due to the .vscode/settings.json we got. However, fixed and pushed already.